Lib/unittest/mock.py

```diff
@@ -2788,8 +2788,8 @@ def create_autospec(spec, spec_set=False, instance=False, _parent=None,
     if _parent is not None and not instance:
         _parent._mock_children[_name] = mock
 
-    wrapped = kwargs.get('wraps')
-
+    # Pop "wraps" from kwargs because it will not be used again.
+    wrapped = kwargs.pop('wraps', None)
```

Comment:
Okay, but what harm does it do being there?
What happens if you revert this change?

Reply:
https://github.com/python/cpython/blob/7c4107be8f482f97eee3086a53c9d06e59a64b92/Lib/unittest/mock.py#L2794-L2796

When `wraps` is given and if it is not popped from `kwargs`, it will make this condition 👇 true when `create_autospec` is called rescursively.

https://github.com/python/cpython/blob/7c4107be8f482f97eee3086a53c9d06e59a64b92/Lib/unittest/mock.py#L2853-L2854

It should be true only for the parent mock because the user will pass configure mock kwargs with respect to the parent attribute.